prow/cluster/jobs/istio/cni/istio.cni.master.yaml

```diff
@@ -113,6 +113,54 @@ postsubmits:
           privileged: true
       nodeSelector:
         testing: test-pool
+  - annotations:
+      testgrid-alert-email: istio-oncall@googlegroups.com
+      testgrid-dashboards: istio-cni-postsubmits-master
+      testgrid-num-failures-to-alert: "1"
```

Comment:
Is this supposed to be a string instead of int? Not sure if the quotes change things.

Reply:
yes, I think because k8s annotations must be strings